subsys/bluetooth/controller/ll_sw/ull_peripheral.c

```diff
@@ -172,6 +180,11 @@ void ull_periph_setup(struct node_rx_hdr *rx, struct node_rx_ftr *ftr,
 		win_delay_us = WIN_DELAY_LEGACY;
 	}
 
+#if (!defined(CONFIG_BT_LL_SW_SPLIT_LLCP_LEGACY))
```

Comment:
nitpick: superfluous ()

Reply:
The conditional compile issues will be fixed after this PR is merged in main